src/components/form-elements/Radio.jsx

```diff
@@ -7,12 +7,15 @@ const Radio = props => {
   const {
     checked,
     name,
+    large,
```

Comment:
I suggest to use 'size' prop and export sizes as const, as followed in many other components

Reply:
changed :)

<img width="1006" alt="screenshot 2018-05-04 10 07 09" src="https://user-images.githubusercontent.com/9083292/39618206-049d2a18-4f83-11e8-99ac-aec71a320959.png">